src/kibana/components/vislib/lib/data.js

```diff
@@ -118,6 +118,9 @@ define(function (require) {
       var data = this.chartData();
       var dataLengths = {};
 
+      //dataLengths.charts = data.filter(function (d) {
+      //  return d.series.length;
+      //}).length;
       dataLengths.charts = data.length;
       dataLengths.stacks = data[i].series.length;
       dataLengths.values = data[i].series[j].values.length;
```

Comment:
It seems to be erroring out here.
![uncaughtexception](https://cloud.githubusercontent.com/assets/549944/8009726/cb096394-0b78-11e5-8362-b65107993818.jpg)